### PR TITLE
output message

### PR DESCRIPTION
**Pull request by 0xMALVEE**
+output update message pointing to installation guide

### CODE REVIEW

File: cli/bin/internal_command_update_prompt.dart [hunk 1]
```diff
@@ -42,5 +42,7 @@ class InternalCommandUpdatePrompt extends InternalCommand {
   @override
   void run(final List<String> args, Context context) {
     shouldAutoUpdateAfterExecutingCommand(args, context);
+    context.printToConsole(
```
Comment (TarasMazepa) [inline cli/bin/internal_command_update_prompt.dart:45]: I changed my mind—this message is no longer needed here. But you need to remove the usage of the `InternalCommandUpdatePrompt.shouldAutoUpdateAfterExecutingCommand` so we would not ask users to update the tool for now. Until we onboard Stax with Brew and Chocolatey fully, and we will be able to check what the current version the user is using and what the most recent one is on the server.

File: cli/bin/internal_command_update.dart [hunk 2]
```diff
@@ -27,5 +27,7 @@ class InternalCommandUpdate extends InternalCommand {
         .announce("Pulling new changes.")
         .runSync()
         .printNotEmptyResultFields();
+    context.printToConsole(
```
Comment (TarasMazepa) [inline cli/bin/internal_command_update.dart:30]: Let's delete all the code in the function except this newly added one. 

Also, in line # 7 of this file, update command to become hidden.

```
InternalCommandUpdate() : super("update", "Updates to the latest version.", type: InternalCommandType.hidden);
```

**Review by TarasMazepa — CHANGES_REQUESTED** [review]
Almost there are some changes needed.

File: cli/bin/internal_command_update_prompt.dart [hunk 3]
```diff
@@ -27,16 +27,8 @@ class InternalCommandUpdatePrompt extends InternalCommand {
...
-        "Stax will update after executing your command.");
-    if (answer) {
-      context.printToConsole(
-          "Thanks for supporting stax and updating it to latest version!");
-    } else {
-      context.printToConsole(
-          "Ok, will ask again in ${silenceDuration.inDays} day(s).");
-    }
-    return answer;
+
```
Comment (TarasMazepa) [inline cli/bin/internal_command_update_prompt.dart:30]: Let's revert changes to this file, and you will be good to go.

**Review by TarasMazepa — APPROVED** [review]

File: cli/bin/internal_command_update_prompt.dart (hunk 3 above)
Comment (0xMALVEE) [reply cli/bin/internal_command_update_prompt.dart]: revert done

**Review by TarasMazepa — APPROVED** [review]

**Pull request merged**